Project/Source/Modules/ModuleInput.cpp

```diff
@@ -78,8 +78,12 @@ UpdateStatus ModuleInput::PreUpdate() {
 				case SDL_WINDOWEVENT_CLOSE:
 					return UpdateStatus::STOP;
 				case SDL_WINDOWEVENT_RESIZED:
-					TesseractEvent e(TesseractEventType::SCREEN_RESIZED);
-					App->events->AddEvent(e);
+					// TODO: Refactor to event
```

Comment:
Event was being created and was substituted by a TODO Refactor to event, meaning?

Reply:
This is because the event was causing lag when resizing the window. It has to be resolved.